students/eric_rosko/lesson-04/mailroom.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python3
+
+'''
+Eric Rosko
+Session03
+mailroom.py
+Tuesday, Nov. 16, 2015
+Description:
+    Part Session 3 homework
+Requirements:
+    You must have py.test installed from http://pytest.org
+    python3 -m pip install pytest
+Usage:
+    py.test -v mailroom.py
+    or
+    python3 mailroom.py
+'''
+
+import json_save_meta as js
+from operator import *
+import io
+
+class DictionarySaver(js.JsonSaveable):
+    dict = js.Dict()
```

Comment:
`dict` is a built in (i.e. type definition) so you need to pick a different var name